src/rgw/rgw_rest_oidc_provider.cc

```diff
@@ -108,21 +69,18 @@ int RGWCreateOIDCProvider::get_params()
       }
   }
 
-  if (provider_url.empty() || thumbprints.empty()) {
-    ldpp_dout(this, 20) << "ERROR: one of url or thumbprints is empty" << dendl;
+  if (thumbprints.empty()) {
+    s->err.message = "Missing required element ThumbprintList";
     return -EINVAL;
   }
 
+  string idp_url = url_remove_prefix(provider_url);
+  resource = rgw::ARN(idp_url, "oidc-provider/", s->user->get_tenant(), true);
```

Comment:
now that the accounts PR has been merged, do we need to pass in the account id/ name here? cc @cbodley

Reply:
yeah i think this needs to rebase on main to pick up those changes